doc/api/stream.md

```diff
@@ -2751,6 +2747,7 @@ contain multi-byte characters.
 [`process.stdout`]: process.html#process_process_stdout
 [`readable.push('')`]: #stream_readable_push
 [`readable.setEncoding()`]: #stream_readable_setencoding_encoding
+[`readable._read()`]: stream.html#stream_readable_read_size_1
```

Comment:
```suggestion
[`readable._read()`]: #stream_readable_read_size_1
```

Reply:
Also: as bottom references are sorted in ASCII order, the undercore needs to be placed before the lower case letters.